### PR TITLE
Text Analytics SDK v4.0.0 - New project structure

### PR DESCRIPTION
**Pull request by assafi**
Please note, this is the successor of #6291 , only difference is the new project structure for the SDKs.

Summary of changes:
- Adding non-async calls
- Adding non-batch calls 
- Correct order of params (id, text, lang)

### CODE REVIEW

**Comment by assafi** [comment]
@dsgouda Could you please review?

**Comment by assafi** [comment]
@dsgouda / @AlexGhiondea - Could one of you please review?

File: sdk/cognitiveservices/Language.TextAnalytics/src/Customizations/TextAnalytics/Models/LanguageInput.cs [hunk 1]
```diff
@@ -27,7 +23,7 @@ public LanguageInput()
         /// Initializes a new instance of the LanguageInput class.
         /// </summary>
         /// <param name="id">Unique, non-empty document identifier.</param>
-        public LanguageInput(string countryHint = default(string), string id = default(string), string text = default(string))
+        public LanguageInput(string id = default, string text = default, string countryHint = default)
```
Comment (AlexGhiondea) [inline sdk/cognitiveservices/Language.TextAnalytics/src/Customizations/TextAnalytics/Models/LanguageInput.cs]: Is this a breaking change?

**Review by AlexGhiondea — CHANGES_REQUESTED** [review]

File: sdk/cognitiveservices/Language.TextAnalytics/src/Customizations/TextAnalytics/Models/LanguageInput.cs (hunk 1 above)
Comment (assafi) [reply sdk/cognitiveservices/Language.TextAnalytics/src/Customizations/TextAnalytics/Models/LanguageInput.cs]: It is, which is why this is a new version.

Comment (AlexGhiondea) [reply sdk/cognitiveservices/Language.TextAnalytics/src/Customizations/TextAnalytics/Models/LanguageInput.cs]: Should the package version be bumped, or did that happen in a different change?

Comment (assafi) [reply sdk/cognitiveservices/Language.TextAnalytics/src/Customizations/TextAnalytics/Models/LanguageInput.cs]: Looks like I forgot to commit the `.csproj` file. Fixed.

**Review by AlexGhiondea — APPROVED** [review]

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Can you link the spec changes you made for this one?
Also your breaking change should follow semantic versioning rules, so with a breaking change, bump up the version to 4.0.0 rather than 3.1.0

Also is this auto generated or you edited this code?

**Comment by assafi** [comment]
@shahabhijeet This change is manual and there are no spec changed to the swagger definition. 
We prefer not update the API definition (which will require a service version bump) just to change the order of parameters in the SDK. The rest of the changes are done manually because there's no support for them in autorest (non-batch and non-async calls). 

I'll update the SDK version to 4.0.0

**Comment by assafi** [comment]
Please note that when auto-generating this SDK from the spec definition the parameter order will be reverted back. The spec itself will be updated on the next version of the service API at which point the correct order of parameters would be fixed. Until then, special attention should be given whenever this SDK is being updated to make sure that the parameter order is not accidentally reverted back.

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**